### PR TITLE
Enable loading ckpts from manifold

### PR DESCRIPTION
**Pull request by ankitade**
Differential Revision: D37988804

### CODE REVIEW

File: torchmultimodal/utils/common.py [hunk 1]
```diff
@@ -96,18 +97,14 @@ def tensor_slice(x: Tensor, begin: List[int], size: List[int]) -> Tensor:
 
 
 def load_module_from_url(
-    model: torch.nn.Module, url: str, progress: bool = True
+    model: torch.nn.Module, url: str, strict: bool = True, progress: bool = True
```
Comment (ebsmothers) [inline torchmultimodal/utils/common.py:100]: nit: pass device?

**Review by ebsmothers — COMMENTED** [review]
One nit, otherwise looks good

**Review by ebsmothers — APPROVED** [review]

File: test/modules/encoders/test_clip_text_encoder.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import torch
 from test.test_utils import assert_expected, set_rng_seed
 from torchmultimodal.modules.encoders.clip_text_encoder import CLIPTextEncoder
+from torchmultimodal.utils.common import load_module_from_url
```
Comment (ebsmothers) [inline test/modules/encoders/test_clip_text_encoder.py:11]: not used?

Comment (ankitade) [reply test/modules/encoders/test_clip_text_encoder.py]: yeah exported again now

File: torchmultimodal/utils/common.py (hunk 1 above)
Comment (ankitade) [reply torchmultimodal/utils/common.py:100]: so device is defaulting to wherever ckpt was saved or cpu if gpu not available.

**Pull request closed without merging**